src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="container" name="personalInfo">
+        <legend>Personal information</legend>
+
+        <label class="block">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="block">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="block">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="block">
+          Full date of birth:
+          <input type="date" name="dataBirth">
+        </label>
+
+        <label class="block">
+          I accept the term of the agreement
+          <input type="checkbox" name="accept">
+        </label>
+
+
+      </fieldset>
+
+      <fieldset class="container" name="registration">
+        <legend>Registration</legend>
+
+        <label class="block">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="block">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="7"
+            maxlength="12"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container" name="interestFact">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="block">
+          Do you love cats?
+          <input
+            type="radio"
+            value="yes"
+            name="loveCats"
+          >
+          Yes
+          <input
+            type="radio"
+            value="no"
+            name="noLoveCats"
+          >
+          No
+        </label>
+
```

Comment:
now click on `Do you love cats` select first radion input.
for fix it wrap each radio input in label, and instead replace label with class block on div with class block

also check and fix this case
![Screenshot from 2023-04-09 17-15-56](https://user-images.githubusercontent.com/33029864/230778086-4237bbbf-46da-49f7-93b8-74375bffc182.png)

Reply:
Done